reducers/views/channel.ts

```diff
@@ -96,7 +98,7 @@ function mobileView(state = false, action) {
 }
 
 // lastUnreadChannel tracks if the current channel was unread and if it had mentions when the user switched to it.
-function lastUnreadChannel(state = null, action) {
+function lastUnreadChannel(state: (Channel & {hadMentions: boolean}) | null = null, action: GenericAction) {
```

Comment:
If we don't have a proper definition for this in the state, we could use this here:
```suggestion
function lastUnreadChannel(state: ({channelId: string; hadMentions: boolean}) | null = null, action: GenericAction) {
```
since this only has the ID and not the whole object

Reply:
Done